README.md

```diff
@@ -1,6 +1,6 @@
 # UI Bridge for Substrate chains.
 
-This is a UI proposal for querying , operating  and testing the bridge.
+This is a UI proposal for querying , operating and testing the bridge.
```

Comment:
```suggestion
This is a UI proposal for querying, operating and testing the bridge.
```

Reply:
i don't understand what changes you are proposing here ... diff looks quite similar

Reply:
the space before comma.